frameworks/Python/uvicorn/app.py

```diff
@@ -84,62 +100,53 @@ def __init__(self, scope):
         pass
 
     async def __call__(self, receive, send):
+        row_id = randint(1, 10000)
```

Comment:
Same comment here as in your other PR.  Pretty sure this should be `randint(1, 10001)`.

Reply:
Same again - randint is inclusive of bounds.